src/helpers/db/populate-api.js

```diff
@@ -16,17 +16,29 @@ async function populateApi(mongo, db) {
     const session = mongo.startSession()
     session.startTransaction()
 
-    const applicants = db.collection('applicants')
-    const newApplicants = await fetchApplicants()
-    await insertData(applicants, newApplicants)
-
-    const applications = db.collection('applications')
-    const newApplications = await fetchApplications()
-    await insertData(applications, newApplications)
-
-    const sites = db.collection('sites')
-    const newSites = await fetchSites()
-    await insertData(sites, newSites)
+    if ((await db.collection('applicants').count()) === 0) {
```

Comment:
Small comment - take it or leave it - but I'd bring this check inside of the initialising function. e.g.

```javascript
async function initialiseData(entities, newEntities) {
  if ((await entities.count()) === 0) { 
    await entities.insertMany(newEntities)
    logger.info(`Inserted ${newEntities.length} ${typeof entities}`)
  }
}
```

Then the rest of code can be like: 

```javascript
      const newApplications = await fetchApplications()
      await initialiseData(db.collection('applications'), newApplications)
```